composer.json

```diff
@@ -1,7 +1,7 @@
 {
   "config": {
     "platform": {
-      "php": "7.4"
+      "php": "8.2"
```

Comment:
I'm not sure about this TBH. There are two places where we can add the platform, one where you can use `||` and one where you cannot. What happens here if we try to run `composer install` in 7.4?

Reply:
Sorry, I thought I replied to this already.

It seems to work okay on my end, when testing it on saluki 7.4. 
Regardless, I changed it to match what we did in [iguana](https://github.com/dxw/iguana/blob/5e2a7f409750d63c6d81c467cae40fc0fb163dbe/composer.json#L13C1-L13C28)